modules/terrain/src/lib/helpers/skirt.js

```diff
@@ -0,0 +1,133 @@
+import {concatenateTypedArrays} from '@loaders.gl/loader-utils';
+
+/**
+ * Add skirt to existing mesh
+ * @param {object} attributes - POSITION and TEXCOOD_0 attributes data
+ * @param {any} triangles - indices array of the mesh geometry
```

Comment:
Should we declare this type as `TypedArray | number[]`?

Reply:
It raises error in `new triangles.constructor` "This expression is not constructable.
  Type 'Function' has no construct signatures.ts(2351)"